pgsync/sync.py

```diff
@@ -1126,8 +1139,8 @@ def sync(
 
             row[META] = Transform.get_primary_keys(keys)
 
-            if i % 100 == 0:
-                logger.debug(f"synced {i} events")
+            if i % batch_size == 0:
+                logger.debug(f"document's batch #{( i // batch_size )}x{batch_size} synced")
```

Comment:
```python
            if i % log_every == 0:
                logger.debug(f"synced {i} documents (batch #{( i // log_every )} x {log_every})")
```
we need `{i}` to know how many documents are synced regardless whether a log_every is 100 or 87634